front/js/shared/yandex.es6

```diff
@@ -19,24 +18,48 @@
     fullEmail: 'info@shopelectro.ru',
   };
 
+  // @todo #129 Implement tracking of certain actions on front-end for YA and GA.
+  //  Actions: one-click purchase, changing products count on order page.
+  //  See the parent issue for a detail.
+
+  // Sync container for yaTracker
+  window.dataLayer = window.dataLayer || [];
+  // Load ecommerce plugin for gaTracker
+  ga('require', 'ecommerce');
+
+  let yaTracker = new YATracker(window.dataLayer, 'RUB');
+  let gaTracker = new GATracker(ga, 'ecommerce');
+
   const init = () => {
     setUpListeners();
+    // e-commerce container
+    // https://yandex.ru/support/metrika/data/e-commerce.html
   };
 
   function setUpListeners() {
     mediator.subscribe('onOneClickBuy', () => {
       reachGoal('CMN_BUY_SEND');
       reachGoal('FAST_BUY_SEND');
     });
-    mediator.subscribe('onOrderSend', () => {
+    mediator.subscribe('onOrderSend', (_, products) => {
       reachGoal('CMN_BUY_SEND');
       reachGoal('FULL_BUY_SEND');
+      // Use a dummy order's id, because we do not wait complete processing of
+      // purchase request.
+      yaTracker.purchase(products, {id: 'DummyId'});
+      gaTracker.purchase({id: 'DummyId'}, products);
```

Comment:
diff args order. Looks like mistake

Reply:
@duker33 It is an issue of refarm-site [ya](https://github.com/fidals/refarm-site/blob/master/ecommerce/front/trackers.es6#L30) [ga](https://github.com/fidals/refarm-site/blob/master/ecommerce/front/trackers.es6#L52)

Reply:
@artemiy312 , so create plz pdd task or refarm-site's task with related comment here